next-tavla/app/(admin)/hooks/useQuaySearch.ts

```diff
@@ -7,17 +7,29 @@ import { useCallback, useEffect, useMemo, useState } from 'react'
 import { countBy } from 'lodash'
 import { getTransportIcon } from 'components/TransportIcon'
 import { useQuery } from 'hooks/useQuery'
+import { TLineFragment } from '../edit/[id]/components/TileCard/types'
 
 function getPlatformLabel(
     index: number,
     publicCode?: string | null,
     description?: string | null,
     directionTypes?: (TDirectionType | null | undefined)[] | null,
+    lines?: TLineFragment[] | null,
 ) {
+    const lineInformation = lines
+        ?.map((line) => line.publicCode)
+        .slice(0, 5)
```

Comment:
hvorfor slicer vi ut en del av linjene?

Reply:
for å ikke se alle linjene, men kan sikkert legge til '...' om det overstiger

Reply:
Ja, slik at man ser at det er flere